pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -176,7 +204,69 @@ msgctxt "#30361"
 msgid "Record max once per day"
 msgstr ""
 
-#empty strings from id 30362 to 30399
+#empty strings from id 30362 to 30364
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30365"
+msgid "1 day"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30366"
+msgid "%d days"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30367"
+msgid "1 week"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30368"
+msgid "%d weeks"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30369"
+msgid "1 month"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30370"
+msgid "%d months"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30371"
+msgid "1 year"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30372"
+msgid "%d years"
+msgstr ""
+
+#. Recording lifetime representation
+#: src/Tvheadend.cpp
+msgctxt "#30373"
+msgid "Until space needed"
```

Comment:
This means that the backend will delete the recording when it needs space for a new recording and we are running out of disk space. The oldest recording tagged as "until space needed" will be deleted first. The naming is quite common for advanced STB's and PVR software.

Tivo uses it, mediaportal, argus tv, and many others.
Here a reference of a normal STB:
https://community.shaw.ca/docs/DOC-1536

- Keep Until: (default - Space Needed) This setting allows you to define if the program should be automatically deleted if the cable box requires space for new recordings. You can leave the setting on Space Needed and the oldest recordings will be removed to free up space for new recordings. Alternatively, you can set this option to I Delete which will require you to delete recordings to free up space or risk interruption of scheduled recordings in the event that the PVR is full.